### DISCUSSION TITLE
Transformation: Calculation should support operations across different query results

### DISCUSSION THREAD

**Discussion by rvcoutinho** [grafana/grafana#38392] [Dashboards, Panels & Visualizations]
**What happened**:
Binary operation does not work on combined data from Postgres and Elasticsearch

**What you expected to happen**:
The binary operation to work

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Grafana version: 7.0.3
- Data source type & version: Mixed (Postgres and Elasticsearch)
- OS Grafana is installed on: Docker version
- User OS & Browser: Ubuntu/Firefox
- Grafana plugins:
- Others:

**Comment by marefr** [comment]
Please include your queries and transformation and screenshot showing what's not working.

**Comment by rvcoutinho** [comment]
![image](https://user-images.githubusercontent.com/1753197/85745548-2a7b3780-b6dc-11ea-904c-3d6c09116739.png)
![image](https://user-images.githubusercontent.com/1753197/85745783-572f4f00-b6dc-11ea-8291-9235bc380513.png)
![image](https://user-images.githubusercontent.com/1753197/85745851-67472e80-b6dc-11ea-8ae3-d9e14503a25d.png)

It basically just works on binary operations from the same source.

**Comment by torkelo** [comment]
It only works on one result set, you have to join the data first. Which is a bit of a problem in your case as you have both time series and a table result

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
